specification/security/resource-manager/Microsoft.Security/preview/2024-07-01-preview/securityConnectors.json

```diff
@@ -0,0 +1,1707 @@
+          }
+        },
+        "tags": [
+          "SecurityConnectors"
+        ],
+        "description": "Updates a security connector",
+        "operationId": "SecurityConnectors_Update",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/securityConnectorName"
+          },
+          {
+            "$ref": "#/parameters/securityConnectorInBody"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SecurityConnector"
+            }
+          },
+          "default": {
+            "description": "Error response that describes why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "x-ms-examples": {
+          "Delete a security connector": {
+            "$ref": "./examples/SecurityConnectors/DeleteSecurityConnector_example.json"
+          }
+        },
+        "tags": [
+          "SecurityConnectors"
+        ],
+        "operationId": "SecurityConnectors_Delete",
+        "description": "Deletes a security connector.",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/securityConnectorName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The resource was deleted successfully"
+          },
+          "204": {
+            "description": "NoContent"
+          },
+          "default": {
+            "description": "Error response that describes why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SecurityConnectorsList": {
+      "type": "object",
+      "description": "List of security connectors response.",
+      "required": [
+        "value"
+      ],
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "The list of security connectors under the given scope.",
+          "items": {
+            "$ref": "#/definitions/SecurityConnector"
+          }
+        },
+        "nextLink": {
```

Comment:
"format": "uri"

Reply:
This version is build on top of the older version with the goal of supporting a new environment only, without the intention to make further unrelated changes.
And an private PR already reviewed and approved by your team, please see it here https://github.com/Azure/azure-rest-api-specs-pr/pull/18430